README.md

```diff
@@ -33,8 +33,6 @@ variable. For detailed instructions for each of our supported platforms, please
         --host [arg]                           Server to connect to
         --port [arg]                           Port to connect to
         --version                              Show version information
-        --verbose                              Increase the verbosity of the output of the shell
-        --quiet                                Silence output from the shell during the connection process
```

Comment:
Not sure about `verbose`, but `quiet` still makes difference in our code, for example the startup messages don't show up, right? If that's the case, we probably should keep these in the documentation 🤔